base_wamas_ubl/lib/wamas/wamas_grammar/auspq.py

```diff
@@ -0,0 +1,190 @@
+        },
+        "IvAusp_HostPosNr": {
+            "type": "int",
+            "length": 6,
+            "dp": False,
+            "dict_key": "RxAusp_HostPosNr",
+            "df_val": False,
+            "df_func": False,
+        },
+        "IvAusp_MId_AId_Mand": {
+            "type": "str",
+            "length": 3,
+            "dp": False,
+            "dict_key": "RxAusp_MId_AId_Mand",
+            "df_val": False,
+            "df_func": False,
+        },
+        "IvAusp_MId_AId_ArtNr": {
+            "type": "str",
+            "length": 20,
+            "dp": False,
+            "dict_key": "RxAusp_MId_AId_ArtNr",
+            "df_val": False,
+            "df_func": False,
+        },
+        "IvAusp_MId_AId_Var": {
+            "type": "str",
+            "length": 5,
+            "dp": False,
+            "dict_key": "RxAusp_MId_AId_Var",
+            "df_val": False,
+            "df_func": False,
+        },
+        "IvMatqk_HMATQ_HMatQ": {
+            "type": "str",
+            "length": 20,
+            "dp": False,
+            "dict_key": False,
+            "df_val": False,
+            "df_func": False,
+        },
+        "IvAusp_LiefMngsWamas_Mng": {
+            "type": "int",
+            "length": 12,
+            "dp": False,
+            "dict_key": "BestMng",
+            "df_val": False,
+            "df_func": False,
+        },
+        "IvAusp_LiefMngsWamas_Gew": {
+            "type": "int",
+            "length": 12,
+            "dp": False,
+            "dict_key": False,
+            "df_val": False,
+            "df_func": False,
+        },
+        "IvArt_HOSTUNITS_HostEinh": {
+            "type": "str",
+            "length": 5,
+            "dp": False,
+            "dict_key": "HostEinheit",
+            "df_val": False,
+            "df_func": False,
+        },
+        "IvArt_Bestand_Einheit": {
+            "type": "str",
+            "length": 5,
+            "dp": False,
+            "dict_key": "HostEinheit",
+            "df_val": False,
+            "df_func": False,
+        },
+        "IvAusp_MId_Charge": {
+            "type": "str",
+            "length": 20,
+            "dp": False,
+            "dict_key": "RxAusp_MId_Charge",
+            "df_val": False,
+            "df_func": False,
+        },
+        "IvAusp_Info": {
+            "type": "str",
+            "length": 40,
+            "dp": False,
+            "dict_key": "RxAusp_Info",
+            "df_val": False,
+            "df_func": False,
+        },
+        "IvAusp_MId_MHD": {
+            "type": "date",
+            "length": 8,
+            "dp": False,
+            "dict_key": False,
+            "df_val": False,
+            "df_func": False,
+        },
+        "LiefSNr": {
+            "type": "str",
+            "length": 20,
+            "dp": False,
+            "dict_key": False,
+            "df_val": False,
+            "df_func": False,
+        },
+        "AUSPQ0051": {
```

Comment:
looks odd no?

Reply:
I saw it's so weird also, but it appeared in this sheet: https://docs.google.com/spreadsheets/d/1Kr8iCgudc7oJixxh097UFjErLpoR1Kts/edit#gid=1216058395

Let me remove it